src/DataStructures/Tensor/Tensor.hpp

```diff
@@ -15,9 +15,17 @@
 #include "DataStructures/DataVector.hpp"
 #include "DataStructures/ModalVector.hpp"
 #include "DataStructures/SpinWeighted.hpp"
+#include "DataStructures/Tensor/Expressions/AddSubtract.hpp"
 #include "DataStructures/Tensor/Expressions/Contract.hpp"
+#include "DataStructures/Tensor/Expressions/Evaluate.hpp"
+#include "DataStructures/Tensor/Expressions/LhsTensorSymmAndIndices.hpp"
+#include "DataStructures/Tensor/Expressions/NumberAsExpression.hpp"
+#include "DataStructures/Tensor/Expressions/Product.hpp"
+#include "DataStructures/Tensor/Expressions/SquareRoot.hpp"
 #include "DataStructures/Tensor/Expressions/TensorAsExpression.hpp"
 #include "DataStructures/Tensor/Expressions/TensorExpression.hpp"
+// Not all the TensorExpression includes are necessary, but we include them so
```

Comment:
Please move this comment to after all the includes. Clang-format won't sort includes across comments, or at least older versions don't

Reply:
Done!